### PR TITLE
feat(sandbox): frontend example of payment-service

### LINKED ISSUE #415: Frontend Example of Payment-Service
Frontend Example of Payment-Service

### PR DESCRIPTION
**Pull request by sumiter92**
## Description

adds example of payment-service(frontend)

Fixes #415 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] Code conforms with the style guide

### CODE REVIEW

File: sandbox/payment-example/frontend/src/CheckoutPage.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+res = (response) => {
+  // handle success
+  if(response){
+    window.location.href = response.request.responseURL;
+  }
+}
+
+handlePayment = (amount,type) => {
+    if( type === 'orders' && this.state.radio === 1){
+      axios.post('http://localhost:3000/place-order-and-pay',{
```
Comment (akshatdubeysf) [inline sandbox/payment-example/frontend/src/CheckoutPage.js:44]: Pick urls from environment, do not hard code.

File: sandbox/payment-example/frontend/src/OrdersPage.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+                    <MDBCardTitle>
+                      <strong>
+                        <a href='#!'>Slim jeans</a>
+                      </strong>
+                    </MDBCardTitle>
+                    <MDBCardText>Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet, consectetur, adipisci.</MDBCardText>
+                    <MDBCardFooter className='px-1'>
+                      <span className='float-left'>9900 ₹</span>
+                      <span className='float-right'>
+                         <MDBBtn gradient="peach" onClick={() => { handleCheckOut(9900) }}>Buy Now</MDBBtn>
```
Comment (akshatdubeysf) [inline sandbox/payment-example/frontend/src/OrdersPage.js:84]: Why not keep the items and their costs as any array/list and create this view dynamically through that list?

File: sandbox/payment-example/frontend/src/SubscriptionPage.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+        <MDBCol lg="4" md="12" className="mb-lg-0 mb-4">
+          <MDBCard
+            className="card-image"
+            style={{
+              backgroundImage:backgroundImageUrl
+            }}
+          >
+            <div className="text-white text-center pricing-card d-flex align-items-center rgba-stylish-strong py-3 px-3 rounded">
+              <MDBCardBody>
+                <h5>Enterprise</h5>
```
Comment (akshatdubeysf) [inline sandbox/payment-example/frontend/src/SubscriptionPage.js:129]: ???

File: sandbox/payment-example/frontend/src/CheckoutPage.js (hunk 1 above)
Comment (sumiter92) [reply sandbox/payment-example/frontend/src/CheckoutPage.js]: done

File: sandbox/payment-example/frontend/src/OrdersPage.js (hunk 2 above)
Comment (sumiter92) [reply sandbox/payment-example/frontend/src/OrdersPage.js]: done

File: sandbox/payment-example/frontend/src/SubscriptionPage.js (hunk 3 above)
Comment (sumiter92) [reply sandbox/payment-example/frontend/src/SubscriptionPage.js:129]: it's just a dummy name of a subscription package plan

**Comment by akshatdubeysf** [comment]
@sumiter92 fix sonar issues

**Review by akshatdubeysf — APPROVED** [review]

**Comment by akshatdubeysf** [comment]
@sumiter92 sonar issues?

**Pull request merged**